Update user guide documentation (#178)
AY2223S1-CS2103T-T10-1/tp: docs/UserGuide.md

| [`home`](#252-navigate-to-home)                               | `home`                                                                  | Navigate to the home page                                                       |
| [`help`](#26-help)                                            | `help`                                                                  | View help                                                                       |
| [`exit`](#27-exiting-the-program)                             | `exit`                                                                  | Exit the program                                                                |

<div markdown="span" class="alert alert-info"> :eye: **See also:**<br/>  
[Peeking at tasks](#224-peeking-at-tasks).
</div>

<div markdown="span" class="alert alert-info"> :information_source: **Note:**
Features marked with * can only be utilised when you are at the home page.
</div>

## 2. Features

<div markdown="block" class="alert alert-info">

**:information_source: Notes about the command format:**<br>

> * You are to include the parameter in the command that are labelled in `UPPER_CASE`.<br>

Reviewer: ```suggestion
* You are to include the parameters in the command that are labelled in `UPPER_CASE`.<br>
```
Grammatical error here?

Author: Gotcha thanks for catching it!

```
find-module CS
```
In either of the above examples, we find every module whose module code starts with CS in Plannit. Here's
a screenshot of Plannit before and after executing the command:

> <<<<<<< HEAD

Reviewer: Looks like you forgot to fully resolve this merge conflict (because HEAD still exists). I found several other instances of this in your code.

Author: You are right. this is weird since I remembered fixing all

| {::nomarkdown}<p align="center"><img src="images/find-module-before.png"/></p><p>Enter the command into the command box and hit <code>ENTER</code>.</p>{:/} | {::nomarkdown}<p align="center"><img src="images/find-module-after.png"/></p> <p align="right"><i>Notice that only modules with module code starting with `CS` are listed!</i><p/>{:/} |

<div markdown="span" class="alert alert-info"> :information_source: **Note:**<br/>
You will remain on the home page after executing the `find-module` command. This is different
from the behavior of [`goto`](#251-navigate-between-modules) command, where the person list will be
updated with the persons associated with the module.
=======
<div markdown="span" class="alert alert-info"> :information_source: **Note:**<br/>You will
remain on the home page after executing the `find-module` command. This is different

> from the behavior of [`goto`](#252-navigate-between-modules-goto) command.

Author: this is 1 of the merge conflict that is unresolved

<div markdown="span" class="alert alert-info">:bulb: **Tip:**<br/>
You may view the tasks added to a module by navigating to a module's page 

> using the [`goto`](##252-navigate-between-modules) command. <br>

Reviewer 2: This `<br>` looks a bit funny on the website, hence why I removed it in the end

Author: Alright! I shall revert it back

Example:
```
goto CS2103T
```
In the above example, we are navigating to the module with module code `CS2103T`. Here's
a screenshot of Plannit before and after executing the command:

| Before executing the command                                                                                                                         | After executing the command                                                                                                                                                                                                                                                  |
|------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | {::nomarkdown}<p align="center"><img src="images/goto-before.png"/></p><p>Enter the command into the command box and hit <code>ENTER</code>.</p>{:/} | {::nomarkdown}<p align="center"><img src="images/goto-after.png"/></p> <p align="right"><i>Notice that the screen is updated with the module's information! <br> (If the task list is not shown, double-click on the module to open it!)</i><p/>{:/} |

Reviewer: Let's discuss about the tone of the wording of this table.

Author: Currently it appears on the table as follow:
Notice that the screen is updated with the module's information!
(If the task list is not shown, double-click on the module to open it!)

Would it be better if I change it to the following?
Notice that the screen is updated with the module's information!
(If the tasks are not shown, double-click on the module to open it!)

Reviewer: I guess we should discuss this - we need a consensus.

Format:  `home`

<div markdown="block" class="alert alert-primary"> :bulb: **Tips:** <br>
You may click on a module to ["peek"](#224-peeking-at-tasks) at a module's task while on the home page. <br> <br>
For more detailed information regarding a specific module, you may navigate to that
module using the [`goto`](#252-navigate-between-modules) command. <br> <br>
The following commands also navigates you back to the home page: <br>
[`add-module`](#211-add-module), [`delete-module`](#212-delete-module), [`add-person`](#241-add-person).
</div>

<div markdown="block" class="alert alert-info"> :information_source: **Note:**
After using the [`goto`](#252-navigate-between-modules) command, executing the `home` command will 
re-enable the following commands: <br>
[`find-module`](#214-find-module), [`list-module`](#215-list-module),
[`find-person`](#246-find-person), [`list-person`](#247-list-person).
</div>

> #### 2.5.2. Navigate between modules

Reviewer 2: ```suggestion
#### 2.5.2. Navigate between modules: `goto`
```
Are we changing the headings?

Author: Not sure yet hence have yet to put it in as of this PR

Format:  `home`

<div markdown="block" class="alert alert-primary"> :bulb: **Tips:** <br>
You may click on a module to ["peek"](#224-peeking-at-tasks) at a module's task while on the home page. <br> <br>
For more detailed information regarding a specific module, you may navigate to that
module using the [`goto`](#252-navigate-between-modules) command. <br> <br>

> The following commands also navigates you back to the home page: <br>

Reviewer: ```suggestion
The following commands also navigate you back to the home page: <br>
```
Grammatical error?

Author: okay ill make the changes!

<div markdown="block" class="alert alert-info">:information_source: **Note:** <br>
You will leave the home page after executing the `goto` command. This is different from the
behavior of [`find-module`](#214-find-module) command. <br> <br>
After using the `goto` command, usage of the following commands will be restricted:<br>
[`find-module`](#214-find-module), [`list-module`](#215-list-module),
[`find-person`](#246-find-person), [`list-person`](#247-list-person). <br> <br>
To re-enable the restricted commands, you may execute any commands that bring you back to the home page
(i.e. [`home`](#251-navigate-to-home)).
</div>

<br>

### 2.6. View help
If you're feeling lost regarding our features, use the `help` command to
open a window with a link that directs you to our user guide!

Format: `help`

> #### Minor enhancement [coming in v2.0] <br>

Reviewer 2: ```suggestion
#### Minor enhancement [coming in v1.5] <br>
```
v1.5?

Author: okay! sounds better!

<div markdown="block" class="alert alert-info">:information_source: **Note:** <br>
You will leave the home page after executing the `goto` command. This is different from the
behavior of [`find-module`](#214-find-module) command. <br> <br>
After using the `goto` command, usage of the following commands will be restricted:<br>
[`find-module`](#214-find-module), [`list-module`](#215-list-module),
[`find-person`](#246-find-person), [`list-person`](#247-list-person). <br> <br>
To re-enable the restricted commands, you may execute any commands that bring you back to the home page
(i.e. [`home`](#251-navigate-to-home)).
</div>

<br>

### 2.6. View help
If you're feeling lost regarding our features, use the `help` command to
open a window with a link that directs you to our user guide!

Format: `help`

#### Minor enhancement [coming in v1.5] <br>

> Stay tuned for Plannit v2.0, where we will be including a table showing all commands

Reviewer: ```suggestion
Stay tuned for Plannit v1.5, where we will be including a table showing all commands
```
Inconsistent version numbering?

Author: oh my missed that. Thanks for flagging it up!
